internal/params/json.go

```diff
@@ -149,7 +152,18 @@ func makeSettingsFromObject(common commonSettings, rawSettings json.RawMessage,
 		return nil, nil, fmt.Errorf("%w: %s", ErrProviderNoLongerSupported, common.Provider)
 	}
 
+	FQDNRegex := "^(?i)(@?|[a-z0-9-]+)(\\.[a-z0-9-]{2,})+$"
+	FQDN := (common.Host + "." + common.Domain)
+	match, _ := regexp.MatchString(FQDNRegex, FQDN)
+
+	if common.Host == "" || common.Domain == "" {
```

Comment:
> The domain can be empty (see issue for examples mentioned). 

Okay, but in #626 you said "All domain strings are not validated for now. It would be good to check at a global scope the domain is: empty" 😂